maven-scm-plugin/src/main/java/org/apache/maven/scm/plugin/UpdateMojo.java

```diff
@@ -90,14 +90,14 @@ public void execute() throws MojoExecutionException {
             if (result instanceof UpdateScmResultWithRevision) {
                 String revision = ((UpdateScmResultWithRevision) result).getRevision();
 
-                getLog().info("Storing revision in '" + revisionKey + "' project property.");
+                getLog().debug("Storing revision in '" + revisionKey + "' project property.");
 
                 if (project.getProperties() != null) // Remove the test when we'll use plugin-test-harness 1.0-alpha-2
                 {
                     project.getProperties().put(revisionKey, revision);
                 }
 
-                getLog().info("Project at revision " + revision);
+                getLog().debug("Project at revision " + revision);
```

Comment:
This is important to leave on INFO for me.

Reply:
reverted